config/settings/base.py

```diff
@@ -70,7 +70,8 @@ def env(name, default=NoDefaultValue, type_=str):
         if default == NoDefaultValue:
             raise ImproperlyConfigured(f"Missing environment variable: {name}.")
         val = default
-    val = type_(val)
+    if val is not None:
+        val = type_(val)
```

Comment:
I just submitted a PR that added an OptionalStr type, essentially, as I'd prefer that not all things have an implicit Optional added to them, but it's not a strong enough preference that I'll stop this.

Reply:
I was wondering about that...but I don't see your PR?

Reply:
That's because I apparently pushed but forgot to make the PR. Here: https://github.com/SFDO-Tooling/MetaShare/pull/241

Reply:
Ah, I found the branch. Hmm, it feels wrong if we need to pass a special type just to configure an optional string.

Reply:
Yeah, I'm not super sold on it. I mean, `Optional[str]` _is_ a type, but this feels clunky; I'd like to be able to compose that type without a special helper function. Another possibility: add an `optional` flag to the `env` function which will preserve `None`?

Reply:
Isn't passing default=None to `env` intended to be the way that you tell `env` a setting is optional?

Reply:
(This might be an easier thing to hash out in person during oddtreat?)

Reply:
It might be! But I think that `default=None` is, now that you put it that way, a nifty way to indicate that it's optional, and I like your PR. Let's go with it!